apps/site/assets/ts/shuttles/components/ShuttlesPage.tsx

```diff
@@ -6,13 +6,18 @@ interface Props {
   route: Route;
 }
 
-const ShuttlesPage = ({ route }: Props): ReactElement<HTMLElement> => (
-  <div className="shuttles__main col-sm-12 col-lg-7">
-    <h1>Shuttle Maps</h1>
-    <ShuttlesOverview places={route.direction_destinations} />
-    <div>Station Detail and dropdown</div>
-    <div>Map2</div>
-  </div>
-);
+const ShuttlesPage = ({ route }: Props): ReactElement<HTMLElement> => {
+  const places =
+    route.id === "Green" ? route.direction_names : route.direction_destinations;
```

Comment:
Would it be reasonable to just always use `direction_names`? (this might be sort of a design question)

Reply:
@ingridpierre ?

Reply:
This change was made in collaboration with @ingridpierre so I'm confident `direction_names` is just needed here in this single case (showing Eastbound/Westbound instead of a concatenation of up to four different GL terminals), while the other pages can show destinations (Park St, Boston College, and so on).